components/event-sources/Makefile

```diff
@@ -53,13 +55,15 @@ vendor: Gopkg.lock
 	@echo '+ Pulling vendored dependencies'
 	@dep ensure -v --vendor-only
 
+$(patched_trace): hack/set-span-id.patch vendor
+	@echo '+ Patching opencensus dependency'
+	@patch -p0 < hack/set-span-id.patch
 
 .PHONY: test
-
-testpkgs := $(shell go list ./... | grep -vE '/apis/|/client/|/cmd/|/testing')
+testpkgs := $(shell go list ./... | grep -vE '/apis/|/client/|/testing')
```

Comment:
There is no test in `cmd`, and `cmd` tests are not very relevant considering we just call `shared.Main` there, so this should remain probably remain ignored.

Reply:
True, not anymore. We were calling sharedmain in one of our early tests. 
Will revert the file.

Reply:
Done